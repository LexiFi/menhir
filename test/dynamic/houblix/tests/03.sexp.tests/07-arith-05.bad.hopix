let x : string = 1 / 2 + 3 * 5 - 6