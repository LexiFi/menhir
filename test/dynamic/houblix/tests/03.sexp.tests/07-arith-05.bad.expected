(((value
    (DefineValue
      (SimpleValue
        (((value (Id x))
           (position ((start_p ("" 1 0 4)) (end_p ("" 1 0 5)))))
          (((value
              (ForallTy ()
                ((value (TyCon (TCon string) ()))
                  (position ((start_p ("" 1 0 8)) (end_p ("" 1 0 14)))))))
             (position ((start_p ("" 1 0 7)) (end_p ("" 1 0 14))))))
          ((value
             (Apply
               ((value
                  (Apply
                    ((value
                       (Variable
                         ((value (Id `-`))
                           (position
                             ((start_p ("" 1 0 31)) (end_p ("" 1 0 32)))))
                         ()))
                      (position ((start_p ("" 1 0 31)) (end_p ("" 1 0 32)))))
                    ((value
                       (Apply
                         ((value
                            (Apply
                              ((value
                                 (Variable
                                   ((value (Id `+`))
                                     (position
                                       ((start_p ("" 1 0 23))
                                         (end_p ("" 1 0 24)))))
                                   ()))
                                (position
                                  ((start_p ("" 1 0 23)) (end_p ("" 1 0 24)))))
                              ((value
                                 (Apply
                                   ((value
                                      (Apply
                                        ((value
                                           (Variable
                                             ((value (Id `/`))
                                               (position
                                                 ((start_p ("" 1 0 19))
                                                   (end_p ("" 1 0 20)))))
                                             ()))
                                          (position
                                            ((start_p ("" 1 0 19))
                                              (end_p ("" 1 0 20)))))
                                        ((value
                                           (Literal
                                             ((value (LInt 1))
                                               (position
                                                 ((start_p ("" 1 0 17))
                                                   (end_p ("" 1 0 18)))))))
                                          (position
                                            ((start_p ("" 1 0 17))
                                              (end_p ("" 1 0 18)))))))
                                     (position
                                       ((start_p ("" 1 0 17))
                                         (end_p ("" 1 0 20)))))
                                   ((value
                                      (Literal
                                        ((value (LInt 2))
                                          (position
                                            ((start_p ("" 1 0 21))
                                              (end_p ("" 1 0 22)))))))
                                     (position
                                       ((start_p ("" 1 0 21))
                                         (end_p ("" 1 0 22)))))))
                                (position
                                  ((start_p ("" 1 0 17)) (end_p ("" 1 0 22)))))))
                           (position
                             ((start_p ("" 1 0 17)) (end_p ("" 1 0 24)))))
                         ((value
                            (Apply
                              ((value
                                 (Apply
                                   ((value
                                      (Variable
                                        ((value (Id `*`))
                                          (position
                                            ((start_p ("" 1 0 27))
                                              (end_p ("" 1 0 28)))))
                                        ()))
                                     (position
                                       ((start_p ("" 1 0 27))
                                         (end_p ("" 1 0 28)))))
                                   ((value
                                      (Literal
                                        ((value (LInt 3))
                                          (position
                                            ((start_p ("" 1 0 25))
                                              (end_p ("" 1 0 26)))))))
                                     (position
                                       ((start_p ("" 1 0 25))
                                         (end_p ("" 1 0 26)))))))
                                (position
                                  ((start_p ("" 1 0 25)) (end_p ("" 1 0 28)))))
                              ((value
                                 (Literal
                                   ((value (LInt 5))
                                     (position
                                       ((start_p ("" 1 0 29))
                                         (end_p ("" 1 0 30)))))))
                                (position
                                  ((start_p ("" 1 0 29)) (end_p ("" 1 0 30)))))))
                           (position
                             ((start_p ("" 1 0 25)) (end_p ("" 1 0 30)))))))
                      (position ((start_p ("" 1 0 17)) (end_p ("" 1 0 30)))))))
                 (position ((start_p ("" 1 0 17)) (end_p ("" 1 0 32)))))
               ((value
                  (Literal
                    ((value (LInt 6))
                      (position ((start_p ("" 1 0 33)) (end_p ("" 1 0 34)))))))
                 (position ((start_p ("" 1 0 33)) (end_p ("" 1 0 34)))))))
            (position ((start_p ("" 1 0 17)) (end_p ("" 1 0 34)))))))))
   (position ((start_p ("" 1 0 0)) (end_p ("" 1 0 34))))))
